pom.xml

```diff
@@ -66,30 +89,30 @@
       <artifactId>guice-multibindings</artifactId>
     </dependency>
     <dependency>
-      <groupId>${jclouds.groupId}</groupId>
+      <groupId>${jclouds.cloudsoft.groupId}</groupId>
       <artifactId>jclouds-core</artifactId>
-      <version>${project.parent.version}</version>
+      <version>${jclouds.cloudsoft.version}</version>
```

Comment:
Are we happy that this refers still to 20160725 and not Duncan's 20160825? I don't think it will really affect the tests, but worth asking.